build.sbt

```diff
@@ -0,0 +1,36 @@
+import Dependencies._
+
+scalaVersion := "2.13.4"
+
+lazy val commonSettings = Seq(
+  resolvers ++= Seq[Resolver](
+    "notify-java-client" at "https://dl.bintray.com/gov-uk-notify/maven/"
+  )
+)
+
+lazy val notifySpi = (project in file("govuk-notify-spi"))
+  .settings(
+    name := "govuk-notify-spi",
+    commonSettings,
+    scalaVersion := "2.13.4",
+    libraryDependencies ++= Seq(
+      keycloakCore,
+      keycloakModelJpa,
+      keycloakServerSpi,
+      mockito % Test,
+      notifyJavaClient,
+      scalaTest % Test,
+      typeSafeConfig
+    ),
+    assemblyJarName in assembly := "govuk-notify-spi.jar",
+    fork in Test := true,
+    envVars in Test := Map(
+      "GOVUK_NOTIFY_API_KEY" -> "testApiKey",
+      "GOVUK_NOTIFY_TEMPLATE_ID" -> "testTemplateId"
+    )
+  )
+
+envVars in Test := Map(
```

Comment:
Duplicate of lines 27-30? Does this need to be defined twice?

Reply:
Fair point